Sources/Tracking/CustomerIO.swift

```diff
@@ -172,6 +176,10 @@ public class CustomerIO: CustomerIOInstance {
             // Only call this code when not possibly being called from a NSE.
             shared.setupAutoScreenviewTracking()
         }
+
+        // Swizzle apns/fcm token methods to register device.
+        Self.shared.swizzleDidReceiveRemoteNotification()
+        Self.shared.swizzleDidReceiveFCMRegistrationToken()
```

Comment:
🚧 

Like we do for auto screenview tracking, I think we need to add to the `SdkConfig` the ability to disable swizzling of register device token.